modules/11_aerosols/exoGAINS/equations.gms

```diff
@@ -8,6 +8,7 @@
 ***--------------------------------------------------------------------------
 *** JeS: factor 0.000001 converts units from M$ to T$: e.g. for sulfur units are [TgS/TWa]*[TWa]*[$/pm_ts]
 
+*** Add up air pollution taxes
```

Comment:
if this comment should be included in the automated model documentation, the line should start with "`*"